crm-contact-exporter.py

```diff
@@ -4,79 +4,92 @@
 from hubspot import HubSpot
 import pandas as pd
 from simple_salesforce import Salesforce
-from sf_login import *
 
-today = date.today()
+import os
+from dotenv import load_dotenv, find_dotenv
 
+load_dotenv(find_dotenv())
+
+
+today = date.today()
 
-def get_contacts_from_zoho():
-    url = "https://accounts.zoho.eu/oauth/v2/token"
 
+def get_zoho_access_token():
     payload = {'grant_type': 'authorization_code',
-               'client_id': '1000.GJW951HCMON3OZ49YLEO1JZC0HCUZY',
-               'client_secret': '0d36b23770308027169350713b92a02e3c6d9efcc1',
-               'redirect_uri': 'http://localhost:8000/test',
-               'code': '1000.cf449a8b14f0429ec8edac6b9cd3e690.ba0a5a149f2975ad4f27d7e51d404cde'}
-    files = []
-    headers = {}
-
-    response = requests.request("POST", url, headers=headers, data=payload, files=files)
+               'client_id': os.environ.get("ZOHO_CLIENT_ID"),
+               'client_secret': os.environ.get("ZOHO_CLIENT_SECRET"),
+               'redirect_uri': os.environ.get("ZOHO_REDIRECT_URI"),
+               'code': os.environ.get("ZOHO_CODE")}
+
+    response = requests.request("POST", os.environ.get("ZOHO_ACCESS_TOKEN_URL"), headers={}, data=payload, files=[])
```

Comment:
It is hard to read. Single line should not contain more then 2 dots '.' or function calls. Remember that, I have already mentioned this.